jest-preset.front.js

```diff
@@ -25,6 +25,10 @@ const moduleNameMapper = {
         path.join(__dirname, 'packages/core/email/admin/src$1'),
         path.join(__dirname, 'packages/plugins/*/admin/src$1'),
       ],
+  '^react$': path.join(__dirname, 'node_modules/react'),
+  '^react-dom$': path.join(__dirname, 'node_modules/react-dom'),
+  '^react-router-dom$': path.join(__dirname, 'node_modules/react-router-dom'),
+  '^styled-components$': path.join(__dirname, 'node_modules/styled-components'),
```

Comment:
Why do we need that? Does it have to do with the unreleased DS version?

Reply:
When we link the DS we want jest to only resolve the local version of the monorepo (because you need to use the same instance for these libs) & not let it resolve to the closest version to the lib code (which is in another folder typically), because of our jest setup you can't use `<rootDir>` which was my preferred way originally.

Reply:
Nice, thanks. Do you think that could be a valuable code-comment? :) Seems like there is value in keeping this.